browses.md

```diff
@@ -54,6 +58,41 @@ Error codes:
 
 **404** if the browse index doesn't exist
 
+## Hierarchical browse index 
+**/api/discover/browses/<:index-name>**
+
+Provide detailed information about a specific hierarchical browse index and access to the parameters required to access the list of items and entries. The JSON response document is as follows
+```json
+{
+  "id": "keyword",
+  "browseType": "hierarchicalBrowse",
```

Comment:
I'm confused by this entry... above there's two `browseType` values listed: `valueList` and `flatBrowse`.  But this has a third value of `hierarchicalBrowse`?  Should this be `valueList`??

UPDATE: Oh, it appears there may be 3 `browseType` values we are proposing?  If so, I think we need to document the differences between all three, perhaps with an example of how each is used.

Does a Hierarchical Browse index still support `/browses/<:index-name>/entries` or `/browses/<:index-name>/items` or both?  If so, is the syntax the same as already documented for these endpoints? If not, then we may need to find a way to restructure this page so that it's clearer how to interact with a `hierarchicalBrowse` vs `valueList` vs `flatBrowse`.  

Currently, it's very unclear to me how these different broweTypes differ because the `browseType` is only briefly mentioned and not referenced again when talking about `/entries` and `/items`.